src/makeRobot.js

```diff
@@ -38,7 +38,78 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      const nameString = `name: ${this.name}, `;
+      const versionString = `chip version: ${this.version}, `;
+      const wheelesString = `wheels: ${this.wheels}`;
+
+      return nameString + versionString + wheelesString;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goRight: function(right = 1) {
+      let rightOn = right;
+
+      if (rightOn < 0) {
+        rightOn = 0;
+      }
+      this.coords.x += rightOn;
```

Comment:
You can make it easier and remove unnecessary variable.
```suggestion
      if (right >= 0) {
        this.coords.x += right;
      }
```